airbyte-commons-worker/src/test/java/io/airbyte/workers/normalization/DefaultNormalizationRunnerTest.java

```diff
@@ -82,7 +85,7 @@ void setup() throws IOException, WorkerException {
         WorkerConstants.DESTINATION_CATALOG_JSON_FILENAME, Jsons.serialize(catalog));
 
     when(processFactory.create(AirbyteIntegrationLauncher.NORMALIZE_STEP, JOB_ID, JOB_ATTEMPT, jobRoot,
-        NormalizationRunnerFactory.BASE_NORMALIZATION_IMAGE_NAME, false, files, null,
+        "airbyte/normalization", false, files, null,
```

Comment:
nitpick: does this intentionally not use the new `NORMALIZATION_IMAGE` constant?

Reply:
Oh! Thanks for noticing. I updated the test first, then added the constant, and didn't add the change to the test. =)